controllers/packetmachine_controller.go

```diff
@@ -203,14 +203,9 @@ func (r *PacketMachineReconciler) reconcile(ctx context.Context, machineScope *s
 			packet.GenerateMachineTag(mUID),
 			packet.GenerateClusterTag(clusterScope.Name()),
 		}
-		if machineScope.IsControlPlane() {
-			tags = append(tags, infrastructurev1alpha3.MasterTag)
-		} else {
-			tags = append(tags, infrastructurev1alpha3.WorkerTag)
-		}
```

Comment:
Why did we remove the tags? Do we not need them?

Reply:
moved it to packet client